test/test-v8debugapi.ts

```diff
@@ -614,9 +638,11 @@ describe('v8debugapi', function() {
           assert.equal(topFrame['function'], 'foo');
           assert.equal(topFrame.locals[0].name, 'n');
           assert.equal(topFrame.locals[0].value, '2');
-          api.clear(bp);
           config.capture.maxFrames = oldMax;
```

Comment:
The same concern about moving this line inside the callback of `api.clear` as described above.